src/components/02_atoms/Widgets/FileUploadWidget.js

```diff
@@ -262,23 +262,36 @@ class FileUploadWidget extends React.Component {
   }
 }
 
-const fileItemSchema = PropTypes.shape({
+const fileItemMultiplePropType = PropTypes.shape({
+  id: PropTypes.toString.isRequired,
+  file: PropTypes.shape({
```

Comment:
Should we at least share the file part of this proptype?

Reply:
Thanks, that is a good idea.